quadtree.js

```diff
@@ -13,11 +13,12 @@ class Point {
 }
 
 class Rectangle {
-  constructor(x, y, w, h) {
+  constructor(x, y, w, h, data) {
     this.x = x;
     this.y = y;
     this.w = w;
     this.h = h;
+    this.userData = data;
```

Comment:
It looks like below you're using the `userData` of each Point to store an id. Is `userData` on a Rectangle part of it that I didn't see?

Reply:
I think this just needs to be removed because it's not being used in the code that I can see.